server/schedule/checker/rule_checker_test.go

```diff
@@ -302,6 +302,31 @@ func (suite *ruleCheckerTestSuite) TestFixLeaderRoleWithUnhealthyRegion() {
 	suite.Nil(op)
 }
 
+func (suite *ruleCheckerTestSuite) TestFixRuleWitness() {
+	suite.cluster.AddLabelsStore(1, 1, map[string]string{"A": "follower"})
+	suite.cluster.AddLabelsStore(2, 1, map[string]string{"B": "leader"})
+	suite.cluster.AddLabelsStore(3, 1, map[string]string{"C": "voter"})
+	suite.cluster.AddLeaderRegion(1, 1, 2)
+
+	suite.ruleManager.SetRule(&placement.Rule{
+		GroupID:   "pd",
+		ID:        "r1",
+		Index:     100,
+		Override:  true,
+		Role:      placement.Voter,
+		Count:     1,
```

Comment:
Please add a test for `Count` larger than 1.

Reply:
Added, thanks!